### ISSUE TITLE
iOS Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by lisbakke** [invertase/react-native-firebase#293]
### Environment

1. Target Platform (e.g. iOS, Android):
**iOS**
2. Development Operating System (e.g. macOS Sierra, Windows 10):
**macOS Sierra**
3. Build tools (Xcode or Android Studio version, iOS or Android SDK version, if relevant):
**Xcode iOS 10**
4. React Native version (e.g. 0.45.1):
RN **0.46.4**
React **16.0.0-alpha.12**
5. RNFirebase Version (e.g. 2.0.2):
**2.0.5**

Steps to reproduce:
1) Create new RN project with react dependencies:
```"react": "^16.0.0-alpha.12", "react-native": "^0.46.4", "react-native-firebase": "^2.0.5"```

2) Setup podfile and run pod install

3) Run.

**What you should see:**
An error like this https://github.com/facebook/react-native/issues/14209

**How I fixed it**
I delete `node_modules/react-native-firebase/ios/RNFirebase.xcodeproj`

**Comment by lisbakke** [comment]
Could that file `react-native-firebase/ios/RNFirebase.xcodeproj` be added to `.npmignore`?

**Comment by chrisbianca** [comment]
@lisbakke I've just read through the issue you linked, and in there you mention that this is due to an `example/demo` folder in the `react-native-firebase` module.  This folder doesn't exist in my node_modules directory, so I don't think this is correct?

We are unable to add `react-native-firebase/ios/RNFirebase.xcodeproj` to `.npmignore` as this would prevent anybody from being able to add the library manually should they need to.

**Comment by lisbakke** [comment]
@chrisbianca It's due to the `RNFirebase.xcodeproj` file. Did you repro the issue? It's pretty bad because it's a very vague issue that takes many hours of debugging to figure out that it's a third party dep.

**Comment by chrisbianca** [comment]
@lisbakke Did you run `react-native link`? This is not supported by RNFirebase as mentioned in the installation instructions.  You will need to unlink RNFirebase and set up as explained in the instructions.

**Comment by D1no** [comment]
I think this is a packanger issue. Is your cli up to date? Here is something similar
https://github.com/storybooks/storybook/issues/1503#issuecomment-317088692

**Comment by Ehesp** [comment]
Closing this issue, if you have any further comments regarding it we'll be happy to open it again.

**Comment by lisbakke** [comment]
@chrisbianca Nope, didn't run `react-native link`. Does this module work for you on RN `0.46.x`?
